Automattic/wp-calypso
# Social Previews: Add Tumblr preview as shared by others

Related to 1204250791794927-as-1204281809108925

## Proposed Changes

This PR implements the Tumblr post preview as shared by others.

## Testing Instructions

### Prerequisites
- Make sure you have a Tumblr account.
- Make sure you have a self-hosted site connected to Jetpack, with a subscription that includes Social (e.g., Jetpack Complete).
- From the _Social_ or _My Jetpack_ page of your site WPadmin, activate Social.
- Then, connect your site to Tumblr (check [these instructions](https://jetpack.com/support/jetpack-social/connecting-to-social-networks/) if you need help).
- Spin up Calypso, by running this branch locally or using a live link below.

### Testing

- Create a new post from your site WPadmin
- In the Jetpack sidebar, make sure the post will be published automatically to your Tumblr page
<img width="395" alt="Screenshot 2023-04-11 at 4 43 17 PM" src="https://user-images.githubusercontent.com/1620183/231283940-732fc281-184d-40f5-9415-22ef3c7c8e53.png">

- Publish it
- Visit `/posts/:site` in Calypso
- You should see the newly created post under the _Published_ tab
- In the post menu, click _Share_, then press the _Preview_ button
<img width="500" alt="Screenshot 2023-04-11 at 4 47 11 PM" src="https://user-images.githubusercontent.com/1620183/231284467-7d3c5008-321b-444f-a3f9-8b732eaa323c.png">

- Select the Tumblr preview
- You'll see a preview of how the post looks when shared by you and someone else



![1](https://user-images.githubusercontent.com/1620183/236022781-7f486fa4-8bcc-4d44-9d2a-fac408eb1f55.png)
![2](https://user-images.githubusercontent.com/1620183/236022783-6df06102-3da1-4898-98a5-7355922a88cc.png)
![3](https://user-images.githubusercontent.com/1620183/236022788-35db4bb0-c9fe-4898-8c6c-4958cd1a86ba.png)



## Pre-merge Checklist




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204281809108919
